src/Val.h

```diff
@@ -1366,7 +1383,9 @@ class VectorVal final : public Val, public notifier::detail::Modifiable {
 		return At(static_cast<unsigned int>(i)).get();
 		}
 
-	unsigned int Size() const { return val.vector_val->size(); }
+	ZAM_vector* RawVector() const	{ return val.vector_val; }
```

Comment:
RawVector seems to be missing documentation :)

Reply:
I've gone through and documented this stuff.